packages/web-components/fast-foundation/src/radio-group/radio-group.ts

```diff
@@ -87,6 +87,16 @@ export class RadioGroup extends FASTElement {
      */
     @attr
     public value: string;
+    protected valueChanged(): void {
```

Comment:
Can you add tests for setting the value of the group and the correct radio being selected?

Reply:
yes, will do, plan to add a few tests for these changes

Reply:
tests added, also for the multiple checked child radio situation